### PR TITLE
Added a warning to caution users about consequences of uninstalling t…

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
Jira issue : [RHDEVDOCS-5496](https://issues.redhat.com/browse/RHDEVDOCS-5496)

Aligned team: DevTools

Pipeline Version(s): 1.10, 1.11, and 1.12

~~OCP Versions(s): enterprise-4.10 and later~~

Docs preview: [Uninstalling the Operator](https://65433--docspreview.netlify.app/openshift-pipelines/latest/install_config/uninstalling-pipelines#op-uninstalling-the-pipelines-operator_uninstalling-pipelines)

SME review: @vdemeester 

QE review: @ppitonak 

Peer review:

### CODE REVIEW

**Comment by masauso-lungu** [comment]
Helllo @vdemeester , This PR is ready for review. And kindly confirm if the [statement here](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/pipelines/uninstalling-pipelines#:~:text=Uninstalling%20only%20the%20Operator%20will%20not%20remove%20the%20OpenShift%20Pipelines%20components%20created%20by%20default%20when%20the%20Operator%20is%20installed.) is still valid.

**Review by vdemeester — APPROVED** [review]

**Review by ppitonak — APPROVED** [review]

File: modules/op-uninstalling-the-pipelines-operator.adoc
Comment (cbippley) [inline modules/op-uninstalling-the-pipelines-operator.adoc:21]: I don't think the sentence needs the commas, based on https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas.

**Review by cbippley — COMMENTED** [review]
One nit, but LGTM! Please wait for @opayne1 to review before requesting a merge review as I'm still in the 3-week training period. Thanks!

File: modules/op-uninstalling-the-pipelines-operator.adoc
Comment (opayne1) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: +1 to Courtney. You could also do
```suggestion
When you uninstall the Operator, all resources in the Pipeline namespace are lost including the secrets you configured with it.
```

**Review by opayne1 — COMMENTED** [review]
Just a small wording suggestion to go along with Courtney's comment.

File: modules/op-uninstalling-the-pipelines-operator.adoc
Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: Noted, I will rework it.

Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc]: Hi @opayne1, I have implemented and resloved the issue as suggested. Thank you.

/label peer-review-needed

**Review by opayne1 — COMMENTED** [review]
lgtm! Just need to squash your commits before moving on to merge review.

**Comment by masauso-lungu** [comment]
Great.
squashing now.

**Comment by masauso-lungu** [comment]
Hello @mramendi , All reviews are done for this PR, kindly advise if merge can be done. thank you.

File: modules/op-uninstalling-the-pipelines-operator.adoc
Comment (mramendi) [inline modules/op-uninstalling-the-pipelines-operator.adoc:21]: it's not "the Pipeline namespace". I believe it is the target namespace where {pipelines-shortname} is installed, by default, the `openshift-pipelines` namespace. We need to verify this with engineering. I have put a question into Slack.

Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: Alright, I see a thumbs up from you @vdemeester. How do you advise?

Comment (mramendi) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: @masauso-lungu I have confirmation from Pavol now - it is the namespace I described

Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: @mramendi Thank you for the update. I will address it accordingly

File: modules/op-uninstalling-the-pipelines-operator.adoc
Comment (mramendi) [inline modules/op-uninstalling-the-pipelines-operator.adoc:21]: typo in namespace name

Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc:21]: Seen it.
Addressed.

Comment (masauso-lungu) [reply modules/op-uninstalling-the-pipelines-operator.adoc]: Can merge request be made now @mramendi ?

**Comment by mramendi** [comment]
@masauso-lungu I have added the request to merge

**Comment by skrthomas** [comment]
Hey @mramendi we no longer cherrypick to enterprise-4.10. Just wanted to confirm that that is OK with you. If not, we can request special permission to do so but in practice we aren't doing it without special permission.

**Comment by mramendi** [comment]
@skrthomas please cherry-pick to pipelines-docs-1.10, pipelines-docs-1.11, pipelines-docs-1.12. Please do not cherry-pick to enterprise branches, that note is outdated.

**Comment by skrthomas** [comment]
merge review lgtm

**Pull request merged**
